### PR TITLE
feat: Refactor tests

### PR DESCRIPTION
**Pull request by beemi**

### CODE REVIEW

File: src/store/actions/initializeAddresses.test.ts [hunk 1]
```diff
@@ -14,17 +14,6 @@ describe('initializeAddresses tests', () => {
...
-      version: '1.0.0',
-    });
-    expect(wallet.state.wallets.length).toBe(1);
-    expect(wallet.state.wallets[0].imported).toBe(true);
-    expect(wallet.state.unlockedAt).not.toBe(0);
-    expect(wallet.state.termsAcceptedAt).not.toBe(0);
-
-    expect(wallet.state.watsNewModalVersion).toBe('1.0.0');
-    expect(wallet.state.keyUpdatedAt).not.toBe(0);
     expect(wallet.state.unlockedAt).not.toBe(0);
```
Comment (monokh) [inline src/store/actions/initializeAddresses.test.ts:17]: the few checks below don't seem necessary to the test

Comment (monokh) [inline src/store/actions/initializeAddresses.test.ts]: the few checks below don't seem necessary to the test

File: src/store/actions/sendTransaction.test.ts [hunk 2]
```diff
@@ -22,68 +23,38 @@ describe('send transaction tests', () => {
...
 
     const walletId = wallet.state.activeWalletId;
     let testnetEnabledAssets = wallet?.state?.enabledAssets?.testnet?.[walletId];
     testnetEnabledAssets = testnetEnabledAssets!.filter((asset) => asset !== 'SOL');
     expect(testnetEnabledAssets?.length).not.toBe(0);
 
-    // initialize addresses for all assets
-    await wallet.dispatch.initializeAddresses({
+    // update balances for one asset
+    await wallet.dispatch.updateBalances({
```
Comment (monokh) [inline src/store/actions/sendTransaction.test.ts]: Is that necessary for the test? I think send works without this

File: src/store/actions/sendTransaction.test.ts [hunk 3]
```diff
@@ -22,68 +23,38 @@ describe('send transaction tests', () => {
...
-    });
   });
 
-  it('should be able to do send transaction using testnet', async () => {
+  it('should be able to do send transaction (AVAX) using testnet', async () => {
     // change network to testnet
     await wallet.dispatch.changeActiveNetwork({
       network: Network.Testnet,
     });
     expect(wallet.state.wallets.length).toBe(1);
```
Comment (monokh) [inline src/store/actions/sendTransaction.test.ts]: Few checks here don't seem relevant to test

File: src/store/actions/sendTransaction.test.ts [hunk 4]
```diff
@@ -108,73 +79,13 @@ describe('send transaction tests', () => {
...
-      to: '0x1234567890123456789012345678901234567890',
-      amount: new BN(10000),
-      data: '',
-      // @ts-ignore
-      fee: 55,
-      gas: 0,
-      feeLabel: FeeLabel.Fast,
-      fiatRate: 10,
-    });
+    const createNotification = jest.fn();
```
Comment (monokh) [inline src/store/actions/sendTransaction.test.ts:72]: you could also pass the mocked function when you setup the walelt
```
wallet = setupWallet({ ...defaultWalletOptions, createNotification: jest.fn(); })
```

**Review by monokh — APPROVED** [review]

File: src/store/actions/initializeAddresses.test.ts (hunk 1 above)
Comment (beemi) [reply src/store/actions/initializeAddresses.test.ts]: Removed

File: src/store/actions/sendTransaction.test.ts (hunk 2 above)
Comment (beemi) [reply src/store/actions/sendTransaction.test.ts]: yes it does works, removed
